.gitlab-ci.yml

```diff
@@ -3,15 +3,36 @@ stages:
 
 include:
   - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-commits.yml'
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-license.yml'
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-github-status-updates.yml'
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-shell-format.yml'
+    file:
+      - '.gitlab-ci-check-commits.yml'
+      - '.gitlab-ci-check-license.yml'
+      - '.gitlab-ci-github-status-updates.yml'
+      - '.gitlab-ci-check-shell-format.yml'
 
 test:check-shell-formatting:
   before_script:
-    - SHELL_SCRIPTS=/dev/null
+    - SHELL_SCRIPTS="src/app src/app-modules/docker-compose tests/app/run.sh tests/app/scenarios/00_basic_ArtifactInstall.run.sh"
 
+test:smoketests:app:
+  stage: test
+  variables:
+    DOCKER_BUILDKIT: 0
+    COMPOSE_DOCKER_CLI_BUILD: 0
+  tags:
+    - docker
+  image: docker:20.10.10-dind-alpine3.14
+  services:
+    - name: docker:20.10.10-dind-alpine3.14
+      alias: docker
+      command: ["--mtu=1440"] # https://gitlab.com/gitlab-com/gl-infra/production/-/issues/5590
+  before_script:
+    - apk --update --no-cache add bash curl aws-cli tree xz jq wget xdelta3
+    - curl -L https://github.com/docker/compose/releases/download/v2.15.1/docker-compose-linux-x86_64 -o /usr/bin/docker-compose
```

Comment:
Can we use image `tiangolo/docker-with-compose` instead, as we do in other places in CI?

Otherwise we risk getting the dependency out of date quickly and unnoticed.

Reply:
I would have to check. I hope it is alpine based, otherwise lots of work.

Reply:
Worth for the sake of maintenance, IMO. Thanks.